draft-ietf-taps-impl.md

```diff
@@ -372,7 +378,9 @@ An implementation may use the Capacity Profile to prefer paths optimized for the
 
 Implementations should process properties in the following order: Prohibit, Require, Prefer, Avoid.
 If Selection Properties contain any prohibited properties, the implementation should first purge branches containing nodes with these properties. For required properties, it should only keep branches that satisfy these requirements. Finally, it should order branches according to preferred properties, and finally use avoided properties as a tiebreaker.
+When ordering branches, an implementation may give more weight to properties that the application has explicitly set than to properties that are default.
 
+As the available protocols and paths on a specific system and in a specific context may vary, the result of sorting and the outcome of racing may vary even given the same Selection and Connection Properties. However, an implementation may aim to provide a consistent outcome to applications, e.g., by preferring protocols and paths that existing Connections with similar Properties are already using.
```

Comment:
I'd still love to say "ought to aim" 

and then (maybe in there implementation) say  are RECOMMENDED to aim...
- I thin k this is a very useful goal, the SHOULD/RECOMMENDED is admittedly a little weak, in that it's hard t precisely conform, but I still think the intent is important to the usefulness of the TAPS system.

Reply:
Sure, "ought to aim" sounds good to me as well.
We don't have normative language in the implementation document, but I'll put a sentence in the API document saying RECOMMENDED.